openedx/stanford/pavelib/utils/i18n_helpers.py

```diff
@@ -115,6 +150,20 @@ def get_theme_dir():
     return get_theme_base_dir(settings.DEFAULT_SITE_THEME)
 
 
+def git_add():
+    sh('git add conf/locale')
+    sh('git add lms/static/js/i18n')
+    subprocess.check_call(
+        'git add conf/locale',
+        cwd=get_theme_dir(),
+        shell=True,
+    )
+    LOG.info(
```

Comment:
I realize the distinction is hazy on log vs print (https://docs.python.org/2/howto/logging.html#when-to-use-logging), to me it felt like these were normal messages you expect to see when running the script so I left them as print. Do you generally just prefer to log everything and avoid print entirely?

Reply:
Yeah, I see the haze.

I think I did do it for consistency. I know I have had issues before
where logs were routed differently, via file redirection and what not.
Having them all go through the same framework, it's a bit easier to work
with.

It seems like if print/log.info are close enough,
I like having it all go through `log` because then it's easy to also
toggle between `log.debug`, `log.warn`, etc. for varying levels of
printing.

I definitely use `print` in scripts, so I'm not consistent yet, but it
seems like if we have the system available/configured and are already
using it, we may as well use everywhere.

:shrug: